File: Nx3CS/bin/test/src/nx3/IBarWidthCalculator.cs

#pragma warning disable 109, 114, 219, 429, 168, 162
namespace nx3{
	public  interface IBarWidthCalculator : global::haxe.lang.IHxObject {
		   double getLeftBarlineWidth(global::nx3.EBarlineLeft barline);
		
		   double getClefWidth(global::nx3.EClef clef);
		
		   double getKeyWidth(global::nx3.EKey key);
		
		   double getTimeWidth(global::nx3.ETime time);
		
		   double getContentLeftMarginWidth(global::nx3.PBar bar);
		
		   double getContentWidth(global::nx3.PBar bar);
		
		   double getBarlineWidth(global::nx3.EBarline barline);
		
		   double getClefsWidth(global::Array<global::nx3.EClef> clef);
		
		   double getKeysWidth(global::Array<global::nx3.EKey> key);
		
	}
}

File: Nx3Java/java/src/nx3/IBarWidthCalculator.java
package nx3;
import haxe.root.*;

@SuppressWarnings(value={"rawtypes", "unchecked"})
public  interface IBarWidthCalculator extends haxe.lang.IHxObject
{
	   double getLeftBarlineWidth(nx3.EBarlineLeft barline);
	
	   double getClefWidth(nx3.EClef clef);
	
	   double getKeyWidth(nx3.EKey key);
	
	   double getTimeWidth(nx3.ETime time);
	
	   double getContentLeftMarginWidth(nx3.PBar bar);
	
	   double getContentWidth(nx3.PBar bar);
	
	   double getBarlineWidth(nx3.EBarline barline);
	
	   double getClefsWidth(haxe.root.Array<nx3.EClef> clef);
	
	   double getKeysWidth(haxe.root.Array<nx3.EKey> key);
	
}


